src/Microsoft.TemplateEngine.Cli/LocalizableStrings.Designer.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.TemplateEngine.Cli {
     [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
     [global::System.Diagnostics.DebuggerNonUserCodeAttribute()]
     [global::System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
-    public class LocalizableStrings {
+    internal class LocalizableStrings {
```

Comment:
Will this be modified back to public next time is regenerated? I think its possible to add something in .csproj to generate this as `internal`

Reply:
@bozturkMSFT do you know if this should be public to be localized?

Reply:
I found this https://stackoverflow.com/questions/4274311/visual-studio-resx-file-default-internal-to-public#4274341

And after looking at Cli.csproj, looks like indeed someone set it to public, I think this can be changed back to default tool

Reply:
yeah, I found this too. I changed generator in the last commits, can revert if localization process needs it to be public.